sandbox/technical_user_guide.md

```diff
@@ -155,19 +151,16 @@ ZmY0OGE1MzdiNTFkYzVkNzgzNGJmNDY2NDE2YTcxNmJkNDUwOGU5MDRh"
 
 **Option 2, which takes advantage of cURL's ability to base-64 encode your clientId and secret**
 
-Format:
-
-```
-curl --user clientId:secret
-```
+With this option, the user can take advantage of cURL's built-in ability to base-64 encode the `clientId` and `secret`, and request and receive their token in a single step.
 
 Example:
 
 ```
-curl --user 3841c594-a8c0-41e5-98cc-38bb45360d3c:f9780d323588f1cdfc3e63e95a8cbdcdd47602ff48a537b51dc5d7834bf466416a716bd4508e904a
+curl -X POST "https://sandbox.bcda.cms.gov/auth/token" --user 3841c594-a8c0-41e5-98cc-38bb45360d3c:f9780d323588f1cdfc3e63e95a8cbdcdd47602ff48a537b51dc5d7834bf466416a716bd4508e904a -H "authorization: Basic Mzg0MWM1OTQtYThjMC00MWU1LTk4Y2MtMzhiYjQ1MzYwZDNjOm\
```

Comment:
Getting closer!
We also need to explicitly indicate that the body is empty, with `-d ''` (see the Option 2 example for reference).
Basically, it would look like `curl -d '' -X POST "https://sandbox.bcda.cms.gov/auth/token" --user ...`

Reply:
Accepted, resolved.